### DISCUSSION TITLE
Empty Result for Searches

### DISCUSSION THREAD

**Discussion by YasiruWickramaratne** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#536] [Q&A]
Hi,

Thanks for the awesome app.

Just want to know why am I getting empty search results for non specific search questions or sentences? 

![image](https://user-images.githubusercontent.com/25566945/134958768-1a8f41b0-09a6-4324-9833-758388af150c.png)

![image](https://user-images.githubusercontent.com/25566945/134959176-a3234e4e-6775-4d23-b4e1-d1710e21b86c.png)

Some searches do show result, ex - Weather and specific searches like president of a country and capital of a country

![image](https://user-images.githubusercontent.com/25566945/134959287-e552a689-9f34-4636-9129-36f72a0b9c72.png)

![image](https://user-images.githubusercontent.com/25566945/134959673-dba9ff25-aaa4-455c-91ac-88adde3b5d44.png)

Is it this because I'm using Windows 11 beta?

**Comment by lucasdeeiroz** [comment]
If you're using the released version, I believe it has an issue showing image results. If you build the app from source, that problem is solved.
